src/bundle/Resources/views/themes/admin/content_type/available_field_types.html.twig

```diff
@@ -17,9 +17,12 @@
     <ul class="ibexa-available-field-types__list">
         {% for item in field_type_toolbar %}
             <li
-                draggable="{{ is_draggable is not defined or is_draggable == true ? 'true' : 'false' }}"
                 class="ibexa-available-field-type"
                 data-item-identifier="{{ item.identifier }}"
+            >
+            <div
```

Comment:
indent?
and <li> is not draggable? how will it work, the li element will stay after drag?

Reply:
in this case, these are field types in right sidebar, this element always stay in its place, as you can add it multiple times